src/main.rs

```diff
@@ -43,6 +45,8 @@ async fn get_adapter() -> Result<(bluer::Adapter, String), String> {
     }
 }
 
+static DEFAULT_SCAN_SERVICE_BEACON: &str = "omnectWifiConfig";
```

Comment:
imho better mv to const variable block at begin of file

Reply:
done.